akka-http-core/src/main/scala/akka/http/impl/engine/client/pool/HostConnectionPool.scala

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.client.pool
+
+import java.util
+
+import akka.NotUsed
+import akka.annotation.InternalApi
+import akka.dispatch.ExecutionContexts
+import akka.event.LoggingAdapter
+import akka.http.impl.engine.client.PoolFlow.{ RequestContext, ResponseContext }
+import akka.http.impl.util.StageLoggingWithOverride
```

Comment:
huh, I forgot we have that.

code info for others:

```
// TODO Try to reconcile with what Akka provides in StageLogging.
// We thought this could be removed when https://github.com/akka/akka/issues/18793 had been implemented
// but we need a few more changes to be able to override the default logger. So for now we keep it here.
```